### PR TITLE
Fixed person national id for foreigners and new  function for compani…

### PR DESCRIPTION
**Pull request by DIOHz0r**
…es' TIN (RIF in Venezuela)

### CODE REVIEW

File: src/Faker/Provider/es_VE/Person.php [hunk 1]
```diff
@@ -152,6 +152,6 @@ public static function suffix()
      */
     public function nationalId()
     {
-        return 'V-'.$this->numberBetween(10000, 100000000);
+        return static::regexify('[VE]-[0-9]{5,9}');
```
Comment (fzaninotto) [inline src/Faker/Provider/es_VE/Person.php]: Regexify is very slow, prefer `numerify` or a sring pattern.

File: src/Faker/Provider/es_VE/Company.php [hunk 2]
```diff
@@ -27,4 +27,13 @@ public static function companyPrefix()
     {
         return static::randomElement(static::$companyPrefix);
     }
+
+    /**
+     * Generate random Taxpayer Identification Number (RIF in Venezuela). Ex J-123456789-1
+     * @return string
+     */
+    public function taxpayerIdentificationNumber()
```
Comment (fzaninotto) [inline src/Faker/Provider/es_VE/Company.php]: You should add a link in the README to mention this locale-specific formatter. Also, a link annotation in this function phpDoc would help.

File: src/Faker/Provider/es_VE/Person.php (hunk 1 above)
Comment (DIOHz0r) [inline src/Faker/Provider/es_VE/Person.php]: what about randomElement + numerify ?... is that better?

Comment (fzaninotto) [inline src/Faker/Provider/es_VE/Person.php]: yes, absolutely

**Comment by fzaninotto** [comment]
Needs rebase.

**Comment by fzaninotto** [comment]
bad rebase, you're embarking commits completely unrelated to your work. Please do a proper rebase on master.

**Comment by DIOHz0r** [comment]
Has been this PR forgotten?

**Comment by fzaninotto** [comment]
no, see my previous comment

**Comment by fzaninotto** [comment]
No news for too long, closing.

**Pull request closed without merging**